### PR TITLE
add support for multiple searches at once

### PR DESCRIPTION
**Pull request by butlerx**
different types of flags
print layout

### CODE REVIEW

**Review by zergless — COMMENTED** [review]
any testing for this yet?

File: rbsearch [hunk 1]
```diff
@@ -4,96 +4,129 @@
...
+# ver 0.4 butlerx - support for more flags and multiple searches at once
 
 SCRIPTNAME="rbsearch"
 EMAIL="bill.gates@hotmail.com"
 USERNAME="microbg"
 FULLNAME="Bill Gates"
+VERSION="0.4"
 
-SEARCH="ldapsearch -D cn=root,ou=ldap,o=redbrick -xLLL -y /etc/ldap.secret"
+SEARCH="ldapsearch -D cn=root,ou=ldap,o=redbrick -xLLL -y /etc/ldap.secret "
```
Comment (zergless) [inline rbsearch]: this space annoys me more than it should

**Comment by butlerx** [comment]
tested and worked

File: rbsearch [hunk 2]
```diff
@@ -4,96 +4,129 @@
...
 # ver 0.3 butlerx - add support for dcu AD
+# ver 0.4 butlerx - support for more flags and multiple searches at once
 
 SCRIPTNAME="rbsearch"
 EMAIL="bill.gates@hotmail.com"
 USERNAME="microbg"
 FULLNAME="Bill Gates"
+VERSION="0.4"
 
 SEARCH="ldapsearch -D cn=root,ou=ldap,o=redbrick -xLLL -y /etc/ldap.secret"
```
Comment (zergless) [inline rbsearch]: :)

**Review by m1cr0man — APPROVED** [review]
LGTM

**Pull request merged**